specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2021-07-01/machineLearningServices.json

```diff
@@ -0,0 +1,4768 @@
+        },
+        "createdOn": {
+          "type": "string",
+          "readOnly": true,
+          "format": "date-time",
+          "description": "The time at which the compute was created."
+        },
+        "modifiedOn": {
+          "type": "string",
+          "readOnly": true,
+          "format": "date-time",
+          "description": "The time at which the compute was last modified."
+        },
+        "resourceId": {
+          "type": "string",
+          "description": "ARM resource id of the underlying compute"
+        },
+        "provisioningErrors": {
+          "type": "array",
+          "description": "Errors during provisioning",
+          "items": {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+          },
+          "readOnly": true
+        },
+        "isAttachedCompute": {
+          "type": "boolean",
+          "description": "Indicating whether the compute was provisioned by user and brought from outside if true, or machine learning service provisioned it if false.",
+          "readOnly": true
+        },
+        "disableLocalAuth": {
+          "type": "boolean",
+          "description": "Opt-out of local authentication and ensure customers can use only MSI and AAD exclusively for authentication."
+        }
+      },
+      "required": [
+        "computeType"
+      ]
+    },
+    "AKS": {
+      "description": "A Machine Learning compute based on AKS.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Compute"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "properties": {
+              "type": "object",
+              "description": "AKS properties",
+              "properties": {
+                "clusterFqdn": {
+                  "description": "Cluster full qualified domain name",
+                  "type": "string"
+                },
+                "systemServices": {
+                  "type": "array",
+                  "description": "System services",
+                  "items": {
+                    "$ref": "#/definitions/SystemService"
+                  },
+                  "readOnly": true
+                },
+                "agentCount": {
+                  "description": "Number of agents",
+                  "type": "integer",
+                  "format": "int32",
+                  "minimum": 0
+                },
+                "agentVmSize": {
+                  "description": "Agent virtual machine size",
+                  "type": "string"
+                },
+                "clusterPurpose": {
+                  "description": "Intended usage of the cluster",
+                  "type": "string",
+                  "default": "FastProd",
+                  "enum": [
+                    "FastProd",
+                    "DenseProd",
+                    "DevTest"
+                  ],
+                  "x-ms-enum": {
```

Comment:
Instead of inheriting from an anonymous object here - could we move the properties in this anonymous object to the `AKS` model?